pkg/virt-launcher/virtwrap/agent-poller/agent_poller_test.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 const (
-	UnitTestPollInitialInterval = 10 * time.Second
+	UnitTestPollInitialInterval = time.Second
```

Comment:
By this you changed the covered scenarios of the tests. Please check what scenarios have been covered before and after this change.

Reply:
I didn't because the tests that don't check the mechanism still dont check it.
Because they have `const interval = 1` 
and the mechanism is trigger if and only if interval > initial interval.

See please https://github.com/kubevirt/kubevirt/pull/5381/files#r606796271
which is the only test that actually check the mechanism.

Reply:
And why you assume all tests here are suppose to check the exponential increase?

Reply:
I don't assume that, and they don't check it actually, I just think that they is no reason they tests will have different initial interval.

Reply:
You changed all tests scenarios instead of taking care the issue for which you suggested this PR.

if you think the current tests scenarios should be changed, reason for it.

Reply:
I didn't actually change them, the change doesn't actually change anything because the interval is 1 second.
Anyhow, updated the PR to affect only the needed test.